roles/grafana/tasks/grafana.yml

```diff
@@ -33,4 +35,8 @@
 - name: insert firewalld rule for grafana
   set_fact:
     firewall_ports: >
-      {{ firewall_ports + [grafana_port] }}
+      {{ firewall_ports }} + [
+        "{{ item }}:{{ grafana_port }}"
+      ]
+  with_items:
+    - "{{ grafana_bind_address }}"
```

Comment:
We shouldn't be using the bind address for firewall rules.  We should be using whichever address clients will be connecting to...